superset/migrations/versions/978245563a02_migrate_iframe_to_dash_markdown.py

```diff
@@ -154,13 +155,13 @@ def upgrade():
                     children.remove(key)
                     children.append(markdown["id"])
 
-                    dashboard.position_json = json.dumps(
-                        position_dict,
-                        indent=None,
-                        separators=(",", ":"),
-                        sort_keys=True,
-                    )
-                    session.merge(dashboard)
+            if keys_to_remove:
+                for key_to_remove in keys_to_remove:
+                    position_dict.pop(key_to_remove)
```

Comment:
nit: as we're not doing anything with the key, perhaps just do
```suggestion
                    del position_dict[key_to_remove]
```